### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by craigbarnes** [richfelker/musl-chartable-tools#3]
Is this code intended to be licensed the same way as musl? If so, would it be possible to add an explicit license statement somewhere in the repo?

**Comment by richfelker** [comment]
Yes, but writing it is a little nontrivial since it requires clarification of the unicode data files which aren't part of the source covered by the license.

**Comment by fosslinux** [comment]
Any chance we could have this added, particularly for the source files?

> it requires clarification of the unicode data files which aren't part of the source covered by the license

FWIW, you could add licensing information to comments in each file, the license doesn't need to cover the entire repository.
